test/typescript/sample-usage.tsx

```diff
@@ -14,14 +14,16 @@ const testArr: ITodo[] = [
   { text: 'mountain', id: '11' },
   { text: 'air', id: '8' },
   { text: 'plants', id: '9' },
+  { text: 'air', id: '8' },
```

Comment:
might as well just remove any changes to this file so we can keep the review smaller